### DISCUSSION TITLE
Cannot delete row from MySQL data source: Error: {{row._id}} does not exist

### DISCUSSION THREAD

**Discussion by owehbeh** [Budibase/budibase#4689] [Help]
The Delete row action that is generated by Budibase app from the data source page generation is not working, it is showing an error that the specified row does not exist.
 
I've tried to check the [documentation ](https://docs.budibase.com/building-apps/design/actions) but it mentions nothing about the delete action rather than it goes the same way as the save action.

Below is the button configuration and the error 
What does the _Row Rev_ and the _{{ [c4ba8c51eeab342968250eff55794e438].[_rev] }}_ come from?

![image](https://user-images.githubusercontent.com/12089028/155687976-fe16549d-fb19-45a6-bfaf-d7f484383ba2.png)

![image](https://user-images.githubusercontent.com/12089028/155688302-b47d865d-1ef8-43ab-954f-3e665cf0919c.png)

I have also tried to change the Row ID from _id to id, did not work as well

**Comment by mjashanks** [comment]
Hey @owehbeh 

Try the following
1. `Row ID` - use your actual primary key column, instead of `_id` - e.g. you might have a `RouteID` column
2. Remove this - i.e. blank value
